tuxemon/core/technique.py

```diff
@@ -107,9 +110,12 @@ def load(self, slug):
 
         # technique use notifications (translated!)
         # NOTE: should be `self.use_tech`, but Technique and Item have overlapping checks
-        self.use_item = T.translate(results["use_tech"])
-        self.use_success = T.translate(results["use_success"])
-        self.use_failure = T.translate(results["use_failure"])
+        try:
```

Comment:
don't wrap everything in a try/except. if "use_tech" fails, then they all fail, even if "use_success" and the rest are fine.
one possible solution would be to implment a function called "maybe_translate", which accepts a string or None.  If it gests a string, it translates it.  If it gets None, it returns an empty string.  Then for each of these call ```
self.use_item = T.maybe_translate(results.get("use_tech"))
```
or something similar.  im not crazy about that b/c we shouldn't allow null/Nones, but we are elsewhere here.

Reply:
Fixed the try statement for the time being.